### PR TITLE
New convenience init with font, color and alignment

### PR DESCRIPTION
**Pull request by icefall**
## Checklist

- [X] New Extension
- [ ] New Test
- [ ] Changed more than one extension, but all changes are related
- [ ] Trivial change (doesn't require changelog)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -54,6 +54,9 @@ All notable changes to this project will be documented in this file.
    - `validateEmail()` in [[PR]](https://github.com/goktugyil/EZSwiftExtensions/pull/429) by *Vic-L*
    - `validateDigits()` in [[PR]](https://github.com/goktugyil/EZSwiftExtensions/pull/429) by *Vic-L*
 
+12. **UILabel**
```
Comment (Khalian) [inline CHANGELOG.md]: 1.10 is already released. Please put this under Unreleased.

File: Sources/UILabelExtensions.swift [hunk 2]
```diff
@@ -15,6 +15,14 @@ extension UILabel {
     public convenience init(x: CGFloat, y: CGFloat, w: CGFloat, h: CGFloat) {
         self.init(x: x, y: y, w: w, h: h, fontSize: 17)
     }
+    
+    /// EZSwiftExtensions
+    public convenience init(font: UIFont, color: UIColor, alignment: NSTextAlignment) {
```
Comment (Khalian) [inline Sources/UILabelExtensions.swift]: Can we write a unit test for this? Also the comment above is incomplete.

**Comment by icefall** [comment]
@Khalian could you please review this again?

**Comment by lfarah** [comment]
I did some minimal fixes.

File: Sources/UILabelExtensions.swift [hunk 3]
```diff
@@ -11,7 +11,20 @@
 import UIKit
 
 extension UILabel {
+
+    /// EZSwiftExtensions: fontSize: 17
+    public convenience init(x: CGFloat, y: CGFloat, w: CGFloat, h: CGFloat) {
+        self.init(x: x, y: y, w: w, h: h, fontSize: 17)
```
Comment (Khalian) [inline Sources/UILabelExtensions.swift]: I am pretty certain I removed this a few commits ago. Lets make the other convenience init default fontSize = 17.

Comment (lfarah) [inline Sources/UILabelExtensions.swift]: So can we remove this? The other one is already 17 by default

Comment (Khalian) [inline Sources/UILabelExtensions.swift]: Yea. The default param would resolve calls without the arg. IIRC I wrote some tests for that.

**Comment by Khalian** [comment]
There are rebase conflicts here. Can someone rebase and resolve conflicts on their private branch and push a squashed singleton commit so that I can merge this?

**Pull request merged**